.github/ISSUE_TEMPLATE/bug.yml

```
@@ -14,9 +14,11 @@ body:
       required: true
   - type: checkboxes
     attributes:
-      description: Please verify that you've followed these steps
+      description: Please verify that you've followed these steps.
       options:
-        - label: The `brew doctor` above contains no "Warning" lines.
+        - label: I did `brew update` and am still able to reproduce my issue.
+          required: true
+        - label: Resolving the warnings from `brew doctor` did not fix my problem.
```

```suggestion
        - label: I have resolved all warnings from `brew doctor` and that did not fix my problem.
```

Would be good to get these into Homebrew/brew after this is merged, too.

---

Will do.